examples/api-platform-demo/admin/zbuild.yml

```diff
@@ -0,0 +1,22 @@
+# syntax=akerouanton/zbuilder:200126-30
+kind: nodejs
+version: 12
+#alpine: true
```

Comment:
should this be removed ?

Reply:
Mh yes and no :smile: I added a `@TODO` comment to remember to uncomment it once Alpine support got merged :wink: